compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_fuchsia.cpp

```diff
@@ -0,0 +1,161 @@
+//===-- sanitizer_symbolizer_fuchsia.cpp
+//-----------------------------------===//
+//
+// Part of the LLVM Proect, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// Implementation of offline markup symbolizer for fuchsia.
+//
+//===----------------------------------------------------------------------===//
+
+#include "sanitizer_platform.h"
+
+#if SANITIZER_SYMBOLIZER_FUCHSIA
+
+#  include <limits.h>
+#  include <unwind.h>
+
+#  include "sanitizer_stacktrace.h"
+#  include "sanitizer_stacktrace_printer.h"
+#  include "sanitizer_symbolizer.h"
+#  include "sanitizer_symbolizer_markup.h"
+
+namespace __sanitizer {
+
+class FuchsiaStackTracePrinter : public StackTracePrinter {
```

Comment:
What differences are there between this and the MarkdownStackTracePrinter? I would think the implementation for those should be the same?

Reply:
MarkdownStackTracePrinter includes a couple functions that are not needed for the FuchsiaStackTracePrinter, specifically the emitting context functions that is why I added a different implementation.